src/components/inputs/datepickerv2/month/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useMonth } from '@datepicker-react/hooks';
+import PropTypes from 'prop-types';
+
+import {
+  Wrapper,
+  MonthLabelContainer,
+  MonthLabel,
+  WeekdayLabel,
+  WeekdaysAndDaysContainer,
+} from './theme';
+import Day from '../day/index';
+
+const Month = ({ year, month, firstDayOfWeek, customWeekdayLabels }) => {
+  const weekdayLabelFormat = (date) => customWeekdayLabels[date.getDay()];
+  const { days, weekdayLabels, monthLabel } = useMonth({
+    year,
+    month,
+    firstDayOfWeek,
+    weekdayLabelFormat,
+  });
+
+  return (
+    <Wrapper>
+      <MonthLabelContainer>
+        <MonthLabel>{monthLabel}</MonthLabel>
+      </MonthLabelContainer>
+
+      <WeekdaysAndDaysContainer>
+        {weekdayLabels.map((dayLabel) => (
+          <WeekdayLabel key={dayLabel}>{dayLabel}</WeekdayLabel>
+        ))}
+      </WeekdaysAndDaysContainer>
+
+      <WeekdaysAndDaysContainer>
+        {days.map((day) => (
```

Comment:
@AnthonyDS-symphony For some reason, the two first values of days array are 0.
To avoid rendering empty components and getting the error `Each child in a list should have a unique "key"` we can change it like this. I assume it has something to do with the firstDayOfWeek prop : https://github.com/tresko/react-datepicker/tree/master/packages/hooks#firstdayofweek-number--undefined-default-1


```suggestion
        {days.map((day) => (
            day.dayLabel && (<Day key={day.dayLabel} date={day.date} day={day.dayLabel} />)
        ))}
```